client/src/pages/Home.js

```diff
@@ -20,10 +20,10 @@ export function Home() {
 					<section className="video-testimonial">
 						<Container>
 							<Row>
-								<Col className=".me-1">
+								<Col sm={12}>
```

Comment:
you can also add the `md=` breakpoint but this is the right direction, good stuff